climpred/prediction.py

```diff
@@ -204,9 +204,4 @@ def compute_uninitialized(
     forecast = forecast.sel(time=common_time)
     reference = reference.sel(time=common_time)
     u = metric(forecast, reference, dim='time', comparison=comparison)
```

Comment:
Just thought I point out that "forecast" and "reference" are spelled out, but "u"nitialized (?) score is just "u" which I assumed to be zonal/u wind at first glance

Reply:
Good call, I'll fix this.